### PR TITLE
Odq 27

### PR DESCRIPTION
**Pull request by AustenCWells**
## Description

[Goes here]

## Todo's before PR

- [ ] Rebase from develop
- [ ] Unit tests passed (`make test`)
- [ ] PR naming according our [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Commits-guideline)
- [ ] PR description describes the overall goals of PR commits
- [ ] Templates been checked with relevant user-story

## Migrations

- [ ] Null fields in models specifies default value
- [ ] You generated and applied migrations (`make migrate`)
- [ ] You updated fixtures (`make export`)

## PR Todo's

Please refer to PR [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-submit-a-pull-request)

- [ ] Specified reviewers (even if it's yourself)
- [ ] Specified assigneses (those who'll merge)
- [ ] Specified labels
- [ ] Specified milestone

## PR review

Please refer to PR review [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-review-a-pull_request)

- [ ] Code syntax formatting checked
- [ ] Debug messages are absent

## PR merge

Select only one:

- [ ] Should be merged?
- [ ] Should be rebased?
- [ ] Should be squashed?

Please refer to PR merge [guide](https://github.com/InformedSolutions/OFS-MORE-DevOps-Tooling/wiki/Pull-requests#how-to-merge-a-pull_request)

### CODE REVIEW

**Comment by jenniferbrown94** [comment]
rerun jenkins build

File: arc_application/views/childminder_views/other_people_in_home.py [hunk 1]
```diff
@@ -337,15 +337,18 @@ def other_people_summary(request):
                         person_model.cygnum_relationship_to_childminder = person_post_data[
                             'cygnum_relationship']
 
-                    person_model.save()
-
                     if person_comments:
```
Comment (jenniferbrown94) [inline arc_application/views/childminder_views/other_people_in_home.py:340]: does this actually flag the health check or just change the status of it? @WitholdP

File: arc_application/views/childminder_views/other_people_in_home.py [hunk 2]
```diff
@@ -337,15 +337,18 @@ def other_people_summary(request):
...
                         section_status = 'FLAGGED'
                         application = Application.objects.get(
                             pk=application_id_local)
                         application.people_in_home_status = section_status
                         application.people_in_home_arc_flagged = True
                         application.save()
+                        person_model.health_check_status = "Started"
```
Comment (jenniferbrown94) [inline arc_application/views/childminder_views/other_people_in_home.py:347]: does this actually flag the health check or just change the status of it? @WitholdP

Comment (witoldpiecewicz) [reply arc_application/views/childminder_views/other_people_in_home.py:347]: It only changes the status

Comment (jenniferbrown94) [reply arc_application/views/childminder_views/other_people_in_home.py:347]: ok - we want it to be flagged as well so the childminder knows they need the adult to redo the health check, we also only want this to happen when the an adult disqualification question is flagged in ARC and not when any of the other adult fields are flagged

**Review by jenniferbrown94 — APPROVED** [review]

**Pull request merged**